### PR TITLE
remove check

### PR DESCRIPTION
**Pull request by mkangia**
a small followup on https://github.com/dimagi/commcare-hq/pull/26836 to remove the check now.

### CODE REVIEW

**Review by orangejenny — APPROVED** [review]

File: corehq/apps/userreports/reports/view.py [hunk 1]
```diff
@@ -346,8 +346,7 @@ def page_context(self):
         if isinstance(self.spec, ReportConfiguration) and self.spec.report_meta.builder_report_type == 'map':
             context['report_table']['default_rows'] = 100
         if self.request.couch_user.is_staff:
-            if not self.data_source._custom_query_provider:
-                context['queries'] = self.data_source.data_source.get_query_strings()
+            context['queries'] = self.data_source.data_source.get_query_strings()
```
Comment (orangejenny) [inline corehq/apps/userreports/reports/view.py]: Didn't notice on the previous PR, but having an attribute `data_source` that itself has an attribute `data_source` is confusing.

Comment (mkangia) [reply corehq/apps/userreports/reports/view.py]: I agree.

Comment (mkangia) [reply corehq/apps/userreports/reports/view.py]: But not going to touch it, because.. i am scared

Comment (orangejenny) [reply corehq/apps/userreports/reports/view.py]: haha, I don't blame you at all

**Pull request merged**